packages/client/base/src/utils/SDKLogger.ts

```diff
@@ -0,0 +1,53 @@
+import { getBrowserLogger } from "../services/logging";
+
+export class SDKLogger {
```

Comment:
If SDKLogger was an abstract class, with abstract methods: `log`, `warn`, and `error`. With two implementations: `DatadogSDKLogger` & `ConsoleSDKLogger`, this would be a little tighter. There's probably no need to have two different logging interfaces / classes (SDKLogger & BrowserLoggerInterface).

Non blocking, either way I think injecting this class is nice improvement.